client/components/tag.jsx

```diff
@@ -0,0 +1,7 @@
+import styles from "../styles/tag.module.css";
+import React from "react";
+export default function Tag({word}) {
+    return (
+          <a className={styles["tag_style"]}>{word}</a>
```

Comment:
In general, we don't want to use the word `style` when naming classes, because it's already assumed that we will add a style to it. You can simply name the class `tag`